code/modules/antagonists/spy/spy_bounty.dm

```diff
@@ -150,33 +152,31 @@
  *
  * * thing - The item to put up on the black market.
  */
-/datum/spy_bounty/proc/send_to_black_market(atom/movable/thing)
+/datum/spy_bounty/proc/send_to_black_market(atom/movable/thing, had_attack_hand_interaction, was_anchored)
```

Comment:
`had_attack_hand_interaction` seems redundant, is there an issue with using `initial`?

Reply:
Also `anchored` and `attack_hand_interaction` do effectively the same thing, I just did both for completeness sake, if there is a problem we can just fall back on using `anchored`

Reply:
Using `initial(item.bitfield)` may interfere with bitflags that have been toggled on runtime.